### PR TITLE
Create sonu.json

### PR DESCRIPTION
**Pull request by s3mant**

### CODE REVIEW

**Review by MaskDuck — DISMISSED** [review]
gud

**Review by MaskDuck — DISMISSED** [review]
gud

File: domains/sonu.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sonu's Website",
+  "repo": "https://github.com/Bad-Boy-Codes/bad-boy-codes.github.io",
+  "owner": {
+    "username": "bad-boy-codes",
+    "email": "sonupanda078@gmail.com"
+  },
+  "record": {
+    "CNAME": "semant.up.railway.app"
```
Comment (TheBotlyNoob) [inline domains/sonu.json:9]: I thought railway broke your code.

**Review by TheBotlyNoob — DISMISSED** [review]

**Review by TheBotlyNoob — APPROVED** [review]
:+1:

**Pull request merged**